### ISSUE TITLE
Try correcting the name to the name of an existing method, or defining a method named

### ISSUE THREAD

**Issue by jaydipgodhani** [creativecreatorormaybenot/wakelock#112]
When I am run the flutter app then shows me an error like below.

`Error output from Xcode build:
↳
    ** BUILD FAILED **

Xcode's output:
↳
    ../../../../.pub-cache/hosted/pub.dartlang.org/wakelock_platform_interface-0.2.0/lib/messages.dart:100:17: Error: The method
    'setMockMessageHandler' isn't defined for the class 'BasicMessageChannel<dynamic>'.
     - 'BasicMessageChannel' is from 'package:flutter/src/services/platform_channel.dart'
     ('../../FlutterSDK/flutter/packages/flutter/lib/src/services/platform_channel.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'setMockMessageHandler'.
            channel.setMockMessageHandler(null);
                    ^^^^^^^^^^^^^^^^^^^^^
    ../../../../.pub-cache/hosted/pub.dartlang.org/wakelock_platform_interface-0.2.0/lib/messages.dart:102:17: Error: The method
    'setMockMessageHandler' isn't defined for the class 'BasicMessageChannel<dynamic>'.
     - 'BasicMessageChannel' is from 'package:flutter/src/services/platform_channel.dart'
     ('../../FlutterSDK/flutter/packages/flutter/lib/src/services/platform_channel.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'setMockMessageHandler'.
            channel.setMockMessageHandler((dynamic message) async {
                    ^^^^^^^^^^^^^^^^^^^^^
    ../../../../.pub-cache/hosted/pub.dartlang.org/wakelock_platform_interface-0.2.0/lib/messages.dart:114:17: Error: The method
    'setMockMessageHandler' isn't defined for the class 'BasicMessageChannel<dynamic>'.
     - 'BasicMessageChannel' is from 'package:flutter/src/services/platform_channel.dart'
     ('../../FlutterSDK/flutter/packages/flutter/lib/src/services/platform_channel.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'setMockMessageHandler'.
            channel.setMockMessageHandler(null);
                    ^^^^^^^^^^^^^^^^^^^^^
    ../../../../.pub-cache/hosted/pub.dartlang.org/wakelock_platform_interface-0.2.0/lib/messages.dart:116:17: Error: The method
    'setMockMessageHandler' isn't defined for the class 'BasicMessageChannel<dynamic>'.
     - 'BasicMessageChannel' is from 'package:flutter/src/services/platform_channel.dart'
     ('../../FlutterSDK/flutter/packages/flutter/lib/src/services/platform_channel.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'setMockMessageHandler'.
            channel.setMockMessageHandler((dynamic message) async {
                    ^^^^^^^^^^^^^^^^^^^^^

    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description`

**Comment by creativecreatorormaybenot** [comment]
Hi @jaydipgodhani, thanks for opening the issue.

Can you post your `flutter doctor -v`?

**Comment by jaydipgodhani** [comment]
@creativecreatorormaybenot  please check `my flutter doctor -v`

```
flutter doctor -v
[✓] Flutter (Channel master, 2.3.0-2.0.pre.206, on macOS 11.0.1 20B29 darwin-x64, locale en-GB)
    • Flutter version 2.3.0-2.0.pre.206 at /Users/Jaydip/_Project_Jaydip/_Flutter_Project/FlutterSDK/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ea09f4ecbc (8 hours ago), 2021-05-14 15:09:02 +0800
    • Engine revision c75ec8ea4d
    • Dart version 2.14.0 (build 2.14.0-115.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.0)
    • Android SDK at /Users/Jaydip/Library/Android/sdk
    • Platform android-30, build-tools 30.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 47.0.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.56.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.22.0

[✓] Connected device (2 available)
    • iPhone 11 (mobile) • 4D2233A0-1ACE-46CE-BE31-7F2394ABD038 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-4 (simulator)
    • Chrome (web)       • chrome                               • web-javascript • Google Chrome 90.0.4430.212
    ! Error: Jaydip is not connected. Xcode will continue when Jaydip is connected. (code -13)
    ! Error: hiren’s iPad is not connected. Xcode will continue when hiren’s iPad is connected. (code -13)
    ! Error: iPhone is not connected. Xcode will continue when iPhone is connected. (code -13)

• No issues found!
```

**Comment by creativecreatorormaybenot** [comment]
@jaydipgodhani It seems `setMockMethodHandler` might have been removed on the latest `master`.

**Comment by creativecreatorormaybenot** [comment]
I will try regenerating the messages to see what the latest code looks like.

**Comment by jaydipgodhani** [comment]
@creativecreatorormaybenot is there any possible solution for that.?

**Comment by creativecreatorormaybenot** [comment]
@jaydipgodhani can you try to use a different channel for the moment?

**Comment by creativecreatorormaybenot** [comment]
@jaydipgodhani I have fixed the issue in `wakelock: ^0.5.2` 🙂

Let me know if you encounter any other problems.
